code/modules/power/apc.dm

```diff
@@ -737,8 +737,11 @@
 		ui.set_autoupdate(state = (failure_timer ? 1 : 0))
 
 /obj/machinery/power/apc/ui_data(mob/user)
+	var/apc_lock = locked
```

Comment:
locked && !(integration_cog && is_servant_of_ratvar(user))

Reply:
That results in the APC being unlocked to everyone.